### PR TITLE
Use `Intl.RelativeTimeFormat`

### PR DESCRIPTION
**Pull request by ddamato**
Satisfies https://github.com/stanchanpsu/whencanipickle/issues/4

Screen Shot 2024-07-16 at 10 40 48 AM

### CODE REVIEW

File: src/pickleball.js [hunk 1]
```diff
@@ -91,6 +91,32 @@ function searchLocation() {
...
+    { amount: 12, name: "months" },
+    { amount: Number.POSITIVE_INFINITY, name: "years" },
+  ];
+
+  let duration = (date - new Date()) / 1000;
+
+  for (let i = 0; i < DIVISIONS.length; i++) {
+    const division = DIVISIONS[i];
+    if (Math.abs(duration) < division.amount) {
+      return formatter.format(Math.round(duration), division.name)
```
Comment (stanchanpsu) [inline src/pickleball.js:122]: Is it possible to capitalize `In`?

**Review by stanchanpsu — APPROVED** [review]
Nice.

**Comment by stanchanpsu** [comment]
Also what IDE do you use? Can you install [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) so formatting doesn't ping ping between us?

File: src/pickleball.js (hunk 1 above)
Comment (ddamato) [reply src/pickleball.js:122]: Updated.

**Comment by ddamato** [comment]
> Also what IDE do you use? Can you install [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) so formatting doesn't ping ping between us?

Prettier causes havoc for me, what is ping ping?

**Comment by stanchanpsu** [comment]
> > Also what IDE do you use? Can you install [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) so formatting doesn't ping ping between us?
> 
> Prettier causes havoc for me, what is ping ping?

I mean so the formatting doesn't change everytime i merge and then you merge. I guess it's fine. Thanks, LTGM!

**Pull request merged**